packages/ra-ui-materialui/src/layout/MenuItemLink.js

```diff
@@ -10,6 +10,8 @@ const styles = theme => ({
         color: theme.palette.text.secondary,
         display: 'flex',
         alignItems: 'flex-start',
+        minHeight: 'min-content',
```

Comment:
According to https://developer.mozilla.org/fr/docs/Web/CSS/width and https://caniuse.com/#search=min-content, this won't work neither on edge nor IE. Can you think of an alternative?

Reply:
Any minHeight would do. I think the height is set at 24px, so setting the min to that value would do the trick as well.

Reply:
Btw, isn't `flexShrink: 0;` enough to resolve this ?

Reply:
@djhi I want to say no, because if it was, the layout bug wouldn't be there.

Reply:
Have you tried to add `flexShrink: 0` to the `root` style ? I had a similar issue on another project and it solved it

Reply:
Any news ?